libvips/foreign/webpsave.c

```diff
@@ -49,16 +71,41 @@
 #endif /*HAVE_CONFIG_H*/
 #include <glib/gi18n-lib.h>
 
+#ifdef HAVE_LIBWEBP
+
+#include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 
 #include <vips/vips.h>
+#include <vips/internal.h>
 
 #include "pforeign.h"
 
-#ifdef HAVE_LIBWEBP
+#include <webp/encode.h>
+#include <webp/types.h>
+#include <webp/mux.h>
+
+typedef int (*webp_import)( WebPPicture *picture,
+        const uint8_t *rgb, int stride );
 
-typedef struct _VipsForeignSaveWebp {
+typedef enum _VipsForeignSaveWebPMode {
```

Comment:
Sorry, this is such a tiny thing, but libvips is using the name `Webp`, not `WebP`, and we can't change it without breaking source compatibility, or introducing a set of compat macros.

Let's have `Webp` for all libvips names, and leave `WebP` for the WebP API calls.